### PR TITLE
Removed the toast on successful tax geolocation [Issue #140]

### PR DESCRIPTION
**Pull request by jusleg**
fixes: #140 
### WHAT kind of change does this PR introduce?
This pr removes the toast upon successful tax geolocation. It was meant for development purposes. Also the gradle file was updated to fix signing properties problems and a specific version of mockito-android was picked. The newest version that came out on april 7 is breaking the tests

### HOW is this accomplished?
  * removed toast line of code
  * updated gradle

### Checklist
- [x] Docs have been added / updated to reflect the changes
- [x] I have reviewed and tested the changes :white_check_mark:
- [x] I have squashed my commits to have a reasonable amount of commits
- [x] All of the code I have written adhere to the coding conventions outlined in the coding conventions section of the wiki

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -91,7 +91,26 @@ dependencies {
     androidTestImplementation 'com.android.support.test:rules:1.0.1'
     androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.1'
     androidTestImplementation 'com.android.support.test.uiautomator:uiautomator-v18:2.1.3'
-    androidTestImplementation 'org.mockito:mockito-android:2.+'
+    androidTestImplementation 'org.mockito:mockito-android:2.16.0'
+}
+
+Properties props = new Properties()
```
Comment (almiche) [inline app/build.gradle]: What is the supposed to be doing ? signingConfig ?

File: app/src/main/kotlin/com/simplemobiletools/calculator/activities/MoneyActivity.kt [hunk 2]
```diff
@@ -162,8 +162,6 @@ class MoneyActivity : SimpleActivity(), Calculator, LocationListener {
                     spawnTaxModal()
                     return
                 }
-                var message: String = String.format("Current Location: " + province + " \n Longitude: " + lastLocation.longitude + " \n Latitude: " + lastLocation.latitude)
```
Comment (almiche) [inline app/src/main/kotlin/com/simplemobiletools/calculator/activities/MoneyActivity.kt]: Clean extraction

**Comment by almiche** [comment]
✅ Styling is adequate and follows coding conventions outlined by the [Kotlin Foundation](https://kotlinlang.org/docs/reference/coding-conventions.html)
  
✅ New methods and functionalities have appropriate tests (testing edge cases + good coverage)

✅ Branch has an acceptable amount of commits with descriptive names and 

✅ PR description matches the code outlined a has mentions to issues

✅ This PR attempts to implement a specific feature (no code blob or epic pr)

Status: **Approved**
Comments: Nice work. Waiting on the answer to my question.

File: app/build.gradle (hunk 1 above)
Comment (jusleg) [reply app/build.gradle]: before that, we couldn't do `./gradlew build` or run the tests in one command. It was working before but I removed it earlier to suppress warnings. It did remove the warnings and didn't break the tests since we are testing unit and instrumentation tests separately. It made it harder to run the full test locally and I think we'll revert to non-parallel tests as travis introduces a lot of flakiness with parallel. Most builds time out or have weird error that we never had before that. This is just to give us the option to swap back and lets us test easily.

File: app/src/main/kotlin/com/simplemobiletools/calculator/activities/MoneyActivity.kt (hunk 2 above)
Comment (jusleg) [reply app/src/main/kotlin/com/simplemobiletools/calculator/activities/MoneyActivity.kt]: As always 💯
